### PR TITLE
Fixed session methods

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: src/Session.php [hunk 1]
```diff
@@ -105,7 +105,7 @@ public function hasSection($section)
 
 	public function getIterator()
 	{
-		return new \ArrayIterator();
+		return new \ArrayIterator(array_keys($this->sections));
```
Comment (enumag) [inline src/Session.php]: This is intended to be compatible with http://api.nette.org/2.2.7/source-Http.Session.php.html#302-318. I didn't test it though.

**Comment by enumag** [comment]
Also I'm not sure about isStarted and exists methods. I had to mock the letter for my tests to work.

```php
		$this->session = Mockery::mock(\Kdyby\FakeSession\Session::class);
		$this->session->shouldReceive('exists')
			->once()
			->andReturn(TRUE);
```

**Comment by fprochazka** [comment]
Thank you

**Comment by fprochazka** [comment]
I think there should be public properties, that will be returned from those methods.

**Comment by enumag** [comment]
@fprochazka Good idea! I don't mean to steal your idea but as I'm writing more tests I already need this feature so I'll send another PR.

**Pull request merged**